test/gtest/ucp/ucp_test.cc

```diff
@@ -857,13 +853,17 @@ void ucp_test_base::entity::fence(int worker_index) const {
 }
 
 void *ucp_test_base::entity::disconnect_nb(int worker_index, int ep_index,
-                                           enum ucp_ep_close_mode mode) {
+                                           uint32_t close_flags)
+{
     ucp_ep_h ep = revoke_ep(worker_index, ep_index);
     if (ep == NULL) {
         return NULL;
     }
 
-    void *req = ucp_ep_close_nb(ep, mode);
+    ucp_request_param_t param;
+    param.op_attr_mask = UCP_OP_ATTR_FIELD_FLAGS;
+    param.flags        = close_flags;
+    void *req          = ucp_ep_close_nbx(ep, &param);
```

Comment:
maybe add helper function to avoid code duplication  above?

Reply:
done